tests/e2e/SymfonyFlex/run_tests.bash

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+if [ "$DRIVER" = "pcov" ]
+then
+    # `pcov` requires at least PHPUnit 8.0 (used by symfony/phpunit-bridge)
+    export SYMFONY_PHPUNIT_VERSION="8.0"
+fi
+
+set -e
+
+run () {
+    local INFECTION=${1}
+    local PHPARGS=${2}
+
+    if [ "$DRIVER" = "phpdbg" ]
+    then
+        phpdbg $PHPARGS -qrr $INFECTION
+    else
+        php $PHPARGS $INFECTION
+    fi
+
+    diff -u -w expected-output.txt infection.log
+}
```

Comment:
SymfonyFlex needs a special environment variable, and it uses old version of PHPUnit which is not compatible with the one we use, so we can't really run this test from `E2ETest`. Adding this file prevents this test from running from  `E2ETest`, and also solves env variable issue.